docs/reference/rest-api/security/create-roles.asciidoc

```diff
@@ -50,6 +50,9 @@ privilege or action.
 `cluster`:: (list) A list of cluster privileges. These privileges define the
 cluster level actions that users with this role are able to execute.
 
+`description`:: (string) Allows providing a brief description of the role.
```

Comment:
we can probably skip the "allows providing" as we can assume all fields are optional unless marked as required.

```suggestion
`description`:: (string) A description of the role.
```